ciao/cmd/root.go

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ciao-project/ciao/ciao/tool"
+	"github.com/ciao-project/ciao/client"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var C client.Client
+
+var CommandFlags = new(tool.CommandOpts)
```

Comment:
What other package needs this?

Reply:
I this the flags should be in this package and also split up for the sub sub commands (like you did for add.)